### PR TITLE
added Query & resolvers for Random Quote and Get Quote By ID

### PR DESCRIPTION
**Pull request by KaleseCarpenter**
Hello, 

Please review when you have time. Thank You. 🙂 @librairica

### CODE REVIEW

**Review by librairica — COMMENTED** [review]
I see some merge conflicts in this branch. Let me know if you need help resolving them!

**Comment by KaleseCarpenter** [comment]
Ok I believe it is fixed. Thank so much for you help. The files that I needed reviewed are the last 3 above your comment and one below: 
- updated schema and query resolver 
-added get By ID
-added comments small changes
-schema update

File: gqlgen/graph/.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		fmt.Println(responseObject.Quote[i].Species.Name)
+	}
+	return responseObject, nil
+
+	/*quote := &model.Quote{
+		ID:     "12-3422-1-23432-009954",
+		Quote:  "I love being my authentic self",
+		Author: "Kalese Carpenter",
+	}
+	return quote, nil*/
```
Comment (librairica) [inline gqlgen/graph/.txt:47]: I totally understand wanting to hold onto the mock, but I recommend deleting comments when possible.

File: gqlgen/graph/schema.graphqls [hunk 2]
```diff
@@ -2,21 +2,18 @@
 #
 # https://gqlgen.com/getting-started/
 
-type Quotes{
+type Quote {
```
Comment (librairica) [inline gqlgen/graph/schema.graphqls:5]: Thanks for changing to singular, I think this is more intuitive.

File: gqlgen/graph/schema.graphqls [hunk 3]
```diff
@@ -2,21 +2,18 @@
...
+  author: String!
 }
 
-"Author of a Quote"
-type Author {
-"The id of the author"
-  id: ID!
-  "The name of the person who created the quote"
-  name: String!
-}
```
Comment (librairica) [inline gqlgen/graph/schema.graphqls:20]: Great simplification.

File: gqlgen/graph/schema.resolvers.go [hunk 4]
```diff
@@ -2,3 +2,76 @@ package graph
...
+func (r *Resolver) Query() generated.QueryResolver { return &queryResolver{r} }
+
+type queryResolver struct{ *Resolver }
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//  - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//    it when you're done.
+//  - You have helper methods in this file. Move them out to keep these resolver files clean.
```
Comment (librairica) [inline gqlgen/graph/schema.resolvers.go:77]:
```suggestion
```

**Review by librairica — APPROVED** [review]
Screen Shot 2022-10-16 at 6 54 23 PM
This is great! I left some suggestions, but the functionality is spot on.

**Pull request merged**